Editor/Scripts/DCCIntegration/Launchers/BlenderLauncher.cs

```diff
@@ -0,0 +1,244 @@
+                    }
+                }
+
+                SetBlenderPath(blenderPath);
+            }
+
+            return blenderPath;
+        }
+
+        public static void SetBlenderPath(string blenderPath)
+        {
+            EditorPrefs.SetString(editorSettingPath, blenderPath);
+        }
+
+        [MenuItem(menuItem_OpenInBlender, priority = 0)]
+        static void OpenBlender()
+        {
+            var selectedAsset = Selection.objects[0];
+            if (selectedAsset == null)
+                return;
+
+            var servers = UnityEngine.Object.FindObjectsOfType<MeshSyncServer>();
+
+            MeshSyncServer server = null;
+
+            foreach (var serverInScene in servers)
+            {
+                if (serverInScene.enabled)
+                {
+                    server = serverInScene;
+                    break;
+                }
+            }
+
+            if (server == null)
+            {
+                server = MeshSyncMenu.CreateMeshSyncServerMenu(null);
+            }
+
+            server.m_DCCAsset = selectedAsset;
+
+            MeshSyncServerInspectorUtils.OpenDCCAsset(server);
+        }
+
+        [MenuItem(menuItem_OpenInBlender, validate = true, priority = 0)]
+        static bool CanOpenBlender()
+        {
+            if (Selection.objects.Length == 1)
+            {
+                string assetPath = AssetDatabase.GetAssetPath(Selection.objects[0]);
+                if (assetPath.EndsWith(".blend"))
+                {
+                    return true;
+                }
+            }
+            return false;
+        }
+
+        private void M_ProcessOnOutputDataReceived(object sender, System.Diagnostics.DataReceivedEventArgs e)
+        {
+            Debug.LogFormat("[BLENDER]\n{0}", e.Data);
+        }
+
+        private void M_ProcessOnErrorDataReceived(object sender, System.Diagnostics.DataReceivedEventArgs e)
+        {
+            Debug.LogErrorFormat("[BLENDER]\n{0}", e.Data);
+        }
+
+        private void M_blenderProcess_Exited(object sender, EventArgs e)
+        {
+            m_blenderProcess = null;
+        }
+
+        #region IDCCLauncher
+
+        public void OpenDCCTool(UnityEngine.Object asset)
+        {
+            Cleanup();
+
+            var startInfo = new System.Diagnostics.ProcessStartInfo();
+            startInfo.FileName = GetBlenderPath();
+
+            var assetPath = AssetDatabase.GetAssetPath(asset).Replace("Assets/", string.Empty);
+            var absoluteAssetPath = Path.Combine(Application.dataPath, assetPath).Replace('\\', '/');
+
+            var scriptPath = Path.GetFullPath("Packages/com.unity.meshsync/Editor/Scripts/DCCIntegration/Launchers/blender_interop.py");
+
+            startInfo.Arguments = $"\"{absoluteAssetPath}\" -P \"{scriptPath}\"";
+
+            if (runMode != RunMode.GUI)
+                startInfo.Arguments = "-b " + startInfo.Arguments;
+
+            if (runMode == RunMode.Background)
```

Comment:
Will we ever need this background mode ?

Reply:
Yes, this is mostly for the bidirectional updates wheere the user does not need to interact with blender.